src/main/scala/is/hail/keytable/KeyTable.scala

```diff
@@ -62,44 +32,40 @@ object KeyTable {
   }
 }
 
-case class KeyTable(@transient hc: HailContext, @transient rdd: RDD[(Annotation, Annotation)],
-  keySignature: TStruct, valueSignature: TStruct) extends Serializable {
-  require(fieldNames.areDistinct())
-
-  private val localValueSignature = valueSignature
-  val (signature, mergeKeyAndValue) = keySignature.merge(localValueSignature)
+case class KeyTable(@transient hc: HailContext, @transient rdd: RDD[Annotation],
```

Comment:
don't need these to be transient anymore, since KeyTable is not serialized (yay!)